index.js

```diff
@@ -194,6 +195,10 @@ var Worker = function (opts) {
         }
     }.bind(this));
 
+    if(opts.shadowfaxFlags){
+        this.shadowfax = () => new shadowfax(opts.shadowfaxFlags)
+    }
```

Comment:
Shadowfax takes care of it. As soon as render completes (successfully or timed out), Shadowfax closes the tab as part of it's cleanup process - see https://github.com/ind9/shadowfax/blob/95f28cf2f92e16fa2dbf052ca7cb7df6d55046bc/src/engine/chrome.ts#L237#L238